core/condition_test.go

```diff
@@ -309,3 +309,12 @@ func TestCondition_evaluateWithConnectedFailure(t *testing.T) {
 		t.Errorf("Condition '%s' should have been a failure", condition)
 	}
 }
+
+func TestCondition_unsetCertificateExpiration(t *testing.T) {
+	condition := Condition("[CERTIFICATE_EXPIRATION] == 0")
+	result := &Result{}
+	condition.evaluate(result)
+	if !result.ConditionResults[0].Success {
+		t.Errorf("Condition '%s' should have been a success", condition)
+	}
+}
```

Comment:
Would you mind adding one more test case with a non-zero value?

The default value would be 0, so just in case, having one more test for non-zero values should be a bit more reliable to prevent future changes from breaking this feature.